app/templates/views/service-settings/edit-service-billing-details.html

```diff
@@ -0,0 +1,35 @@
+
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/form.html" import form_wrapper %}
+
+{% from "components/textbox.html" import textbox %}
+
+{% block service_page_title %}
+  Change billing details
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header(
+    'Change billing details',
+    back_link=url_for('main.service_settings', service_id=current_service.id)
+  ) }}
+
+  {% call form_wrapper() %}
+    {{ form.billing_contact_names(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.billing_contact_email_addresses(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.billing_reference(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ form.purchase_order_number(param_extensions={"classes": "govuk-!-width-full"}) }}
+    {{ textbox(
+      form.notes,
+      rows=4,
+      width='1-1',
+      autofocus=True,
```

Comment:
You probably don’t want focus jumping to the last field on the page
```suggestion
      autofocus=False,
```